cpp/modules/luma.gl/core/src/model.cpp

```diff
@@ -70,19 +75,27 @@ void Model::draw(wgpu::RenderPassEncoder pass) {
   pass.Draw(this->vertexCount, 1, 0, 0);
 }
 
-void Model::_initializeVertexState(ComboVertexStateDescriptor& cVertexState,
-                                   const std::shared_ptr<garrow::Schema>& attributeSchema) {
-  cVertexState.vertexBufferCount = attributeSchema->num_fields();
+void Model::_initializeVertexState(utils::ComboVertexStateDescriptor* descriptor,
+                                   const std::shared_ptr<garrow::Schema>& attributeSchema,
+                                   const std::shared_ptr<garrow::Schema>& instancedAttributeSchema) {
+  // TODO(ilija@unfolded.ai): Iterate over these two separately if number of fields is expected to be large
+  std::vector fields = attributeSchema->fields();
+  if (instancedAttributeSchema != nullptr) {
```

Comment:
I think `if (instancedAttributeSchema)` should be enough... if we prefer that...